src/classes/console/Console.js

```diff
@@ -0,0 +1,251 @@
+const { Console } = require("console");
+const Colors = require("./Colors");
+const moment = require("moment");
+const { inspect } = require("util");
+
+/**
+ * Komada's console class, extends NodeJS Console class.
+ *
+ */
+class KomadaConsole extends Console {
+
+  /**
+   * Constructs our KomadaConsole instance
+   * @param  {boolean}  [stdout=process.stdout] The location of standard output. Must be a writable stream.
+   * @param  {boolean}  [stderr=process.stderr] The location of standrad error outputt. Must be a writable stream.
+   * @param  {boolean} [colors={}] The colors for this console instance.
+   * @param  {boolean}  [timestamps=false] Whether or not Timestamps should be enabled.
+   */
+  constructor({ stdout, stderr, useColor, colors = {}, timestamps = false }) {
+    super(stdout, stderr);
+    /**
+     * The standard output stream for this console, defaulted to process.stderr.
+     * @name KomadaConsole#stdout
+     * @type {WritableStream}
+     */
+    Object.defineProperty(this, "stdout", { value: stdout });
+
+    /**
+     * The standard error output stream for this console, defaulted to process.stderr.
+     * @name KomadaConsole#stderr
+     * @type {WritableStream}
+     */
+    Object.defineProperty(this, "stderr", { value: stderr });
+
+    /**
+     * Whether or not timestamps should be enabled for this console.
+     * @type {boolean}
+     */
+    this.timestamps = timestamps;
```

Comment:
(Por from Klasa):
```javascript
this.timestamps = timestamps === true ? "YYYY-MM-DD HH:mm:ss" : timestamps;
```

Reply:
why the value & type equality?

Unless it's to strictly enforce Boolean?